Solutions/Windows Security Events/Analytic Rules/MultipleFailedFollowedBySuccess.yaml

```diff
@@ -65,5 +65,5 @@ entityMappings:
     fieldMappings:
       - identifier: Address
         columnName: IPCustomEntity
-version: 1.0.1
+version: 1.0.2
 kind: Scheduled
```

Comment:
Again we should update the Account entity here

Reply:
Done